docs/pages/guides/in-app-purchases.mdx

```diff
@@ -50,12 +50,15 @@ The following libraries provide robust support for in-app purchase functionality
 />
 
 <BoxLink
-  title={
-    <>
-      <CODE>react-native-iap</CODE>
-    </>
-  }
-  description="A React Native library providing an interface to the client-side native in-app purchase API's for Android (both Play Store and Amazon) and iOS."
-  href="https://github.com/dooboolab-community/react-native-iap"
+  title={<><CODE>react-native-iap</CODE> (Deprecated)</>}
+  description="A legacy React Native library for in-app purchases. Superseded by expo-iap for Expo users."
```

Comment:
```suggestion
  description={<>A legacy React Native library for in-app purchases. Superseded by <CODE>expo-iap</CODE> for Expo apps.</>}
```

- Let's add inline syntax highlight for the library name. Also, since this library is not maintained, can you add a warning callout in the library's readme and its docs to indicate the same?
- "Expo apps" is more preferable than "users".

Also, I wonder if you'd like to update reactnative.directory for the same: https://reactnative.directory/?search=iap